gallery/gallery/lib/layout/highlight_focus.dart

```diff
@@ -67,6 +70,7 @@ class _HighlightFocusState extends State<HighlightFocus> {
 
     return Focus(
       canRequestFocus: widget.hasFocus,
+      debugLabel: widget.debugLabel,
```

Comment:
What is the debug label for?

Reply:
It identifies nodes in the output of debugDumpFocusTree()